asaplib/pca/ml_pca.py

```diff
@@ -1,51 +1,135 @@
 """
 tools for doing PCA
 e.g.
-eva = np.genfromtxt(prefix+".desc")
-proj = pca(eva,6)
+desc = np.genfromtxt(prefix+".desc")
+proj = PCA(pca_d).fit_transform(desc)
 """
 
 import numpy as np
 import scipy.linalg as salg
+from sklearn.preprocessing import StandardScaler
 
```

Comment:
Would it make sense to use a library implementation of PCA here e.g. the scikit-learn implementation of PCA? The pros and cons of both custom vs. library implementations are discussed here:

https://www.quora.com/Should-I-write-my-own-code-or-use-libraries  

Additional factors to consider may be how familiar the user is likely to be with the scikit-learn implementation of PCA viz. a custom implementation. In terms of extensibility it's worth considering what functionalities are envisaged to be needed in the future and whether the user will need to add these themselves to a custom implementation.

Reply:
For our purpose, it's neccesaty to have custom PCA as we need to project atomic environemnts

Reply:
Which components of this PCA class are the ones specific to atomic environments?